developer_manual/examples/migrations/Version20161130090952.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace OCA\MyAppName\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20161130090952 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
```

Comment:
I suggest adding a real example here which would ease the life of many.

Have a look at https://github.com/owncloud/core/pull/26923/files#diff-f853a252a02574b7da2d70ca20e06a6eR18 for table creation (ignore `hasTable` for now).

Note that we also need to clarify whether we want developers to always explicitly specify the correct collation as it's not automatic. See this issue: https://github.com/owncloud/core/issues/26925

I'll try and clarify this with @DeepDiver1975 next week